x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterQueryIT.java

```diff
@@ -370,14 +353,7 @@ protected void testSearchesAgainstNonMatchingIndices(boolean exceptionWithSkipUn
         {
             String q = "FROM cluster-a:nomatch";
             String expectedError = "Unknown index [cluster-a:nomatch]";
-            expectVerificationExceptionForQuery(q, expectedError, requestIncludeMeta);
-        }
-
-        // an error is thrown if there are no matching indices at all - single remote cluster with wildcard index expression
-        {
-            String q = "FROM cluster-a:nomatch*";
-            String expectedError = "Unknown index [cluster-a:nomatch*]";
-            expectVerificationExceptionForQuery(q, expectedError, requestIncludeMeta);
+            expectVerificationExceptionForQuery(q, expectedError, requestIncludeMeta);// TODO this must contain the remote cluster alias
```

Comment:
Here we are getting `RemoteTransportException` with cause `IndexNotFoundException` with index `nonmatch`. It is correct, but it does not give me enough information to point out to `cluster-a` in case there are multiple remotes involved. This is likely needs to be added on FieldCaps side